ctms/crud.py

```diff
@@ -105,6 +105,18 @@ def _contact_base_query(db):
     )
 
 
+def get_all_contacts(db):
+    """Fetch all contacts."""
+    bulk_contacts = _contact_base_query(db)
+    return bulk_contacts.order_by(asc(Email.email_id)).all()
+
+
+def get_all_contacts_from_ids(db, email_ids):
```

Comment:
I don't see a better way to do this right now and it's outside the scope of this PR, but I don't love that we have to create a new function every time we want to filter the `_contact_base_query` in a specific way. We also have a `get_bulk_contacts` crud function which sounds like it'd be useful here, but it has a pretty specific job.